perllib/FixMyStreet/Cobrand/UK.pm

```diff
@@ -328,6 +328,21 @@ sub report_check_for_errors {
         );
     }
 
+    # can't put this in the relevant cobrand as it won't then apply on national site
+    unless ( $errors{name} ) {
+        if ( $report->to_body_named('Buckinghamshire') && length($report->name) > 50 ) {
+            $errors{name} = sprintf( _('Names are limited to %s characters in length.'), 50 );
```

Comment:
Don't think we need translations here, unless these counties are planning their own localisations :) I guess it doesn't matter from a site performance POV but would save putting extra strings in `FixMyStreet.po`.